src/objectsAdvanced/task2.js

```diff
@@ -0,0 +1,8 @@
+const fn = (object) => {
+    const { title } = object;
```

Comment:
It's fine, you made it. Just you could improve your solution by dont using const at all, just extracting needed fields inside function arguments definition (move { title } instead of object. Try to understand how it works. You call a function, passing and object, but function description says: hey, I need only one field from that object!